src/coreclr/jit/emitarm64.cpp

```diff
@@ -8048,6 +8114,216 @@ void emitter::emitIns_R_R_R(
             fmt = IF_DV_3A;
             break;
 
+        case INS_sve_and:
```

Comment:
perhaps I should also let the tool generate this code?

Reply:
That would be useful, even if it's just some isVectorRegister() asserts